BETA_TESTING.md

```diff
@@ -1,47 +1,66 @@
-# Beta testing
+# Build DevTools
 
-This page describes the fastest way to test the latest version of DevTools with all experemental features enabled. Do not mix this setup with development environment.
+This page describes the fastest way to build DevTools with the goal to use it. Do not mix this setup with development environment. If you want to make code changes, follow [contributing guidance](https://github.com/flutter/devtools/blob/master/CONTRIBUTING.md).
 
-If you want to make code changes, follow [contributing guidance](https://github.com/flutter/devtools/blob/master/CONTRIBUTING.md).
+You may want to build DevTools to:
 
-The steps were tested for Mac. They may require adjustments for other platforms.
+1. Try experimental features
+
+2. Run desktop version, instead of Web version, to get get rid of browser memory limit. For example,
+to be able to analyze heap snapshots of more complicated applications.
+
+The steps were tested for Mac and may require adjustments for other platforms. Contributions,
+that make the steps more platform agnostic, are welcome.
 
 ## Prerequisites
 
 1. [Configure](https://docs.flutter.dev/get-started/install) Dart or Flutter.
 
-## Setup and start
+## Setup DevTools
+
+If it is initial setup:
 
 1. In your terminal `cd` to a folder where you want to clone devtools, and that does not have subfolder `devtools` yet.
 
-2. Clone the repo, get needed Flutter version to local folder and start DevTools:
+2. Clone the repo and get needed Flutter version to local folder:
 
 ```bash
 git clone https://github.com/flutter/devtools.git
 
-./devtools/tool/update_flutter_sdk.sh
-
-cd devtools/packages/devtools_app
-../../tool/flutter-sdk/bin/flutter run -d chrome --dart-define=enable_experiments=true
+bash devtools/tool/update_flutter_sdk.sh
+cd devtools
 ```
 
-3. Paste the URL of your application (for example [Gallery](https://github.com/flutter/devtools/blob/master/CONTRIBUTING.md#connect-to-application)) to the connection textbox.
+## Or refresh DevTools
 
-## Refresh and start
+If you are refreting previously configured version:
```

Comment:
"If you have already configured your DevTools environment and need to refresh to get the latest DevTools code, follow these instructions."